src/lando/try_api/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class TryConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
+    name = "try"
```

Comment:
```suggestion
    name = "lando.try"
```

Conventionally this is also named after the module (`try_api` in this case). Any reason to deviate from that?